lib/streamlit/hello/__init__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018-2019 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import streamlit.hello
+
+if __name__ == "__main__":
+    streamlit.hello.run()
```

Comment:
Why is this needed? We could just have `cli.py` call `run()`, right?

Reply:
Yeah,... +1

Reply:
I did ended up removing this and refactoring the imports.